src/parser/clauses/index.c

```diff
@@ -0,0 +1,10 @@
+#include "./index.h"
+#include "../ast_common.h"
+
+AST_IndexNode* AST_IndexOp(const char *label, const char *property, AST_IndexOpType optype) {
+  AST_IndexNode *indexOp = malloc(sizeof(AST_IndexNode));
+  indexOp->target.label = label;
+  indexOp->target.property = property;
```

Comment:
In my current testing, it has not proven necessary to `strdup` these inputs, but I could be overlooking something? Let me know if I should change this!

Reply:
You're right, we don't have to duplicate these.